libs/application-generic/src/usecases/upsert-preferences/upsert-preferences.usecase.ts

```diff
@@ -61,12 +61,12 @@ export class UpsertPreferences {
     });
   }
 
-  private async upsert(
+  public async upsert(
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Only the `execute` method should be `public` in use-cases. We should revert this change and use `execute` as needed by passing in the necessary command properties to control the use-case behaviour.‏
```suggestion
  private async upsert(
```

Reply:
I see that other methods are `public` in the `UpsertPreferences` - I will address those in a follow-up PR, but we should avoid increasing the problem for now.

Reply:
Oh, I was using upsert directly at some point then I removed that implementation and forgot to put the method back to private.